CareKit/CareCard/OCKCareCardButton.m

```diff
@@ -53,6 +55,16 @@ - (void)drawRect:(CGRect)rect {
         [[UIColor clearColor] setFill];
         UIRectFill(_circleLayer.frame);
     }
+    if (!_label) {
+        CGRect labelRect = CGRectMake(0, ButtonSize/2 + 8, ButtonSize, ButtonSize);
```

Comment:
I think creating a static const of value 8, with an explanatory variable name will be better.